src/service/userService.js

```diff
@@ -314,6 +314,42 @@ function deleteOsUser (userDetails, callback) {
   return registryService.updateRecord(regReq, callback);
 }
 
+async function transferAssetOfDeleteduser(req, response){
+  logObject['message'] = userMessages.DELETE.OWNERSHIP_TRANSFER
+  logObject['traceId'] = req.headers['x-request-id'] || '',
+  loggerService.entryLog(req.body, logObject);
+  try {
+  KafkaService.sendRecordWithTopic(req.rspObj.request, envVariables.COKREAT_USER_DELETE_KAFKA_TOPIC, function (err, res) {
+    if (err) {
+      handelAssetOwnershiptransfer(req, err);
```

Comment:
The method name is not proper. You are not having the positive scenario here.

User the right method name lik "owershipTransferErrorHandler"